R/parsed.R

```diff
@@ -1,3 +1,15 @@
+#' Parse and pre-process character vector
+#'
+#' @description The function obtains detailed parse information for `text` via
+#'   [utils::getParseData] and does some minimal pre-processing by calling
+#'   [add_ws_to_parse_data].
+#' @inheritSection add_ws_to_parse_data Details
```

Comment:
Well `compute_parse_data_flat_with_ws` is the complement of `serialize_parse_data_flat`. I thought about merging `add_ws_to_parse_data` with `compute_parse_data_flat_with_ws` since the latter calls the former (see line 13 to 18 in `parsed.R`. I would not merge the complements.